gpm_api/tests/conftest.py

```diff
@@ -94,6 +103,7 @@ def remote_filepaths() -> Dict[str, Dict[str, Any]]:
             "start_time": datetime.datetime(2020, 7, 5, 17, 0, 44),
             "end_time": datetime.datetime(2020, 7, 5, 18, 33, 17),
             "version": 7,
+            "granule_id": 36092,
```

Comment:
Do you think this should be refactor or extended? Or generated from specific functions ?

Reply:
Each entry has a little difference to test various things. Maybe the entries could be indeed templated to look less copy pasted :)

Reply:
Note this down and let's improve this at later stage ;)